doc/api/url.md

```diff
@@ -1151,13 +1151,22 @@ console.log(url.domainToUnicode('xn--iñvalid.com'));
 // Prints an empty string
 ```
 
-### `url.fileURLToPath(url)`
+### `url.fileURLToPath(url[, options])`
 
 <!-- YAML
 added: v10.12.0
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/52509
+    description: The `options` argument can now be used to
+                 determine how to parse the `path` argument.
```

Comment:
Nit: `fileURLToPath` doesn't parse `path` argument, it produces a path.

Reply:
*(resolved locally)*